collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1790,7 +1790,7 @@ private void createQuitDialog() {
 		   FormController formController = Collect.getInstance().getFormController();
 		   title = (formController == null) ? null : formController.getFormTitle();
 		   if ( title == null ) {
-		      title = "<no form loaded>";
+		      title = "<".concat(getString(R.string.no_form_loaded)).concat(">");
```

Comment:
I would tend to put the <> in the string resource. I don't feel strongly about it but I think it does make the code a bit cleaner.